app/database/db_enums.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+)
+
+var (
+	enumsMutex  sync.RWMutex
+	enumColumns = []string{
+		"permissions_granted.can_view",
+		"permissions_granted.can_grant_view",
+		"permissions_granted.can_watch",
+		"permissions_granted.can_edit",
+		"group_managers.can_manage",
+	}
+	enumName2Number     map[string]int
+	enumValueName2Index []map[string]int
+	enumValueIndex2Name []map[int]string
+)
+
+func loadDBEnum(db *DB, fullColumnName string) {
```

Comment:
why not `func (db *DB) loadEnum(fullColumnName string) ` ? (same for other functions below)

Reply:
fixed